openapscontrib/timezones/__init__.py

```diff
@@ -35,13 +53,16 @@ def get_program (self, args):
     return program
   def set_converter (self, args):
     params = self.get_params(args)
-    self.FIELDNAME = params.get('field')
+    self.FIELDNAME = params.get('date')
     self.adjust = params.get('adjust')
     self.timezone = params.get('timezone')
+    self.astimezone = params.get('astimezone')
 
   def rezone (self, dt):
     if (self.adjust == 'missing' and dt.tzinfo == None) or self.adjust == 'replace':
-      return dt.replace(tzinfo=self.timezone)
+      dt = dt.replace(tzinfo=self.timezone) # .astimezone(self.timezone)
```

Comment:
This replaces the timezone, such that the "datetime" expressed is now in this new timezone rather than the old one.

When a datetime that does not know about a timezone gains the ability to reason about timezones, calling `isoformat( )` on the object will result in the similar behavior as javascript's `dt.toISOString( )`: the date will be rendered as iso 8601, in Zulu/UTC timezone, with the hours shifted from the local timezone appropriately.

The following `dt.astimezone` line, in contrast, "shifts" the hours from timezone A to timezone B appropriately while pinning the behavior of `isoformat( )` to produce an iso8601 format in the new timezone (not Zulu/UTC).

These two behaviors can be invoked by the tool independently, to control whether or not to shift times vs simply printing them in local time vs UTC time.